c7n/resources/s3.py

```diff
@@ -1308,7 +1308,55 @@ def process_bucket(self, bucket):
             s3.put_bucket_policy(Bucket=bucket['Name'], Policy=json.dumps(p))
         return {'Name': bucket['Name'], 'State': 'PolicyRemoved', 'Statements': found}
 
+       
+@actions.register('remove-bucket-replication')
```

Comment:
we've stopped doing add/remove in favor of set-x action to better express goal state convergence instead of operations

Reply:
Ok! In this case, would using unset make sense then? unset-bucket-replication?

Reply:
its more like

```
type: set-bucket-replication
state: disabled
```

with the intent that it can also be used to turn on or update replication by passing in a set of rules

Reply:
Ok cool, thanks for clarifying like this. It makes sense to me now. I'll update to reflect this pattern.